### PR TITLE
feat: add comment downvote reputation events

### PR DESCRIPTION
**Pull request by capJavert**
- reputation workers for comment downvote

### CODE REVIEW

File: src/common/pubsub.ts [hunk 1]
```diff
@@ -505,6 +509,26 @@ export const notifyReputationIncrease = async (
     userAfter,
   });
 
+export const notifyCommentDownvoted = async (
```
Comment (idoshamun) [inline src/common/pubsub.ts:512]: I'd suggest to use the type safe version we have now for workers and messages. not a blocker but ideally as we move forward new implementations should utilize that

**Review by idoshamun — COMMENTED** [review]
Please see my comment and let me know how you'd like to proceed

File: src/common/pubsub.ts (hunk 1 above)
Comment (capJavert) [reply src/common/pubsub.ts:512]: Yep, I'll move, also 👍 for implementing new workers.

Comment (capJavert) [reply src/common/pubsub.ts]: be0b580ed2de42e5c010dccd863ca5df4e228d08

File: src/workers/cdc/primary.ts [hunk 2]
```diff
@@ -317,6 +317,51 @@ const handleCommentUpvoteChange = async (
...
+          userId: data.payload.after.userId,
+        });
+      }
+      break;
+    }
+    case 'd': {
+      const wasDownvoted = data.payload.before.vote === UserVote.Down;
+
+      if (wasDownvoted) {
+        await triggerTypedEvent(logger, 'api.v1.comment-downvote-canceled', {
```
Comment (rebelchris) [inline src/workers/cdc/primary.ts:352]: Just checking but with the typed ones did we not say we always send the entity as a whole to make it more streamlined for future additions? 
(can't fully recall if that was the decision)

**Review by rebelchris — APPROVED** [review]
Looks alright, one non blocking question

File: src/workers/cdc/primary.ts (hunk 2 above)
Comment (capJavert) [reply src/workers/cdc/primary.ts:352]: I think at the end it was more of a make it constant inside the subscriptions of the same topic. The [current typed ones ](https://github.com/dailydotdev/daily-api/blob/0c623e2911702484d5a3b0887071c7f177170142/src/common/typedPubsub.ts#L10)are also defined like that.

**Pull request merged**